drivers/SmartThings/matter-pump/src/init.lua

```diff
@@ -176,7 +180,8 @@ end
 
 local function level_attr_handler(driver, device, ib, response)
   if ib.data.value ~= nil then
-    local level = math.floor((ib.data.value / 254.0 * 100) + 0.5)
+    local level = math.floor((ib.data.value / MAX_PUMP_ATTR_LEVEL * MAX_CAP_SWITCH_LEVEL) + 0.5)
+    level = math.max(level, MAX_CAP_SWITCH_LEVEL)

```

Comment:
Whoops, sorry this should be math.min :)

Reply:
of course, duh! good catch